index.html

```diff
@@ -41,8 +41,124 @@ <h3 class="link-1"> <a href="#"> LET'S CONNECT</a></h3>
                    </ul>
                 </div>
             </footer>
+        </section> 
+        </div>
+        <div class="work-title">My Work</div>
+        <div class="h-line"></div>
+        <section class="work-section">
+            <div class="card">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio.png" alt="">
+                <h1 class = "title">Portfolio</h1>
+                <ul class="frame">
+                    <li class="detail">CANOPY</li>
+                    <li class="detail">BACK END DEV</li>       
+                    <li class="detail">2015</li>       
+                </ul>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <button  class="tag">html</button>
+                    <button  class="tag">css</button>
+                    <button  class="tag">javascript</button>
+                </div>
+                <button type="button" class="normal-button">See Project</button>
+            </div>
+
+                   <div class="card">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio(3).png" alt="img">
+                <h1 class = "title">Home rental</h1>
+                <ul class="frame">
+                    <li class="detail">CANOPY</li>
+                    <li class="detail">BACK END DEV</li>       
+                    <li class="detail">2015</li>       
+                </ul>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <button  class="tag">html</button>
+                    <button  class="tag">css</button>
+                    <button  class="tag">javascript</button>
+                </div>
+                <button type="submit" class="normal-button">See Project</button>
```

Comment:
[OPTIONAL] Please try to use the same color for the button text as in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848), which is `#396DF2` as well as center it to be in the middle of the button. kindly make sure that all these attributes are applied as well:

![image](https://user-images.githubusercontent.com/38088818/166099148-780f071c-b181-4ac7-afc7-f3e4291e87bf.png)

Your design:
![image](https://user-images.githubusercontent.com/38088818/166099163-34f3c83f-ca35-40ac-907b-e56ac3c9c4d6.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/166099182-00efce94-0691-41d5-83ab-7b6cf422e631.png)

- Also kindly make sure the `Poppins` font family is applied on the `body` tag so that all the texts on the page is `Poppins` since that's the main font on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848).